### PR TITLE
Upstream v5.47.0

### PR DESCRIPTION
**Pull request by t0yv0**
Fixes https://github.com/pulumi/pulumi-aws/issues/3870

Upgrades upstream to [v5.47.0](https://github.com/hashicorp/terraform-provider-aws/releases/tag/v5.47.0).

Previously this provider was using a patched version of terraform-plugin-framework as motivated by https://github.com/pulumi/terraform-plugin-framework/commit/4d4aedf99afbfb4878d1a5eb829b45035ca29dd6 but the patch conflicts with the latest version (1.8.0). It appears that the patch is no longer necessary.

### CODE REVIEW

**Comment by t0yv0** [comment]
Investigating the upgrade test failure for RDS Instance. It appears that this change has introduced a dedicated_log_volume parameter:

https://github.com/hashicorp/terraform-provider-aws/commit/e5736dc923cd5b71178dd8cb9c92b1324b98f42e

Therefore upgrading to the new version of the provider generates an update plan to move from missing value to the new default of `false`. This should be benign as it is not a replace plan, however the upgrade test is overfitting a bit.

I will weaken the upgrade test to accept this.

File: scripts/tidy-all.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+git ls-files '**go.mod' | xargs -L 1 ./scripts/tidy.sh
```
Comment (VenelinMartinov) [inline scripts/tidy-all.sh:5]: neat, I had a local script for this!

File: scripts/upgrade_upstream.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```
Comment (VenelinMartinov) [inline scripts/upgrade_upstream.sh:1]: Thanks for adding these, is this added through ci-mgmt?

Comment (t0yv0) [reply scripts/upgrade_upstream.sh:1]: Not yet, since scripts are free-floating. Can elevate to Makefile targets at some point if useful.

**Pull request merged**